rackcity/api/urls.py

```diff
@@ -1,6 +1,7 @@
-from rackcity.api.views import ArticleViewSet
-from rest_framework.routers import DefaultRouter
+from django.urls import path
+from rackcity import views
 
-router = DefaultRouter()
-router.register(r'', ArticleViewSet, basename='rackcity')
-urlpatterns = router.urls
\ No newline at end of file
+urlpatterns = [
+    path('models/', views.model_list),
+    path('models/<int:pk>/', views.model_detail),
+]
```

Comment:
2 new url paths (navigate to `localhost:8000/api/models/` or `localhost:8000/api/models/#/`